risc0/circuit/rv32im/README.md

```diff
@@ -1,2 +1,20 @@
 
-The RISC Zero zkVM circuit
\ No newline at end of file
+The RISC Zero zkVM circuit
+
+# Crate Feature Flags
+
+The following crate feature flags are available. They are configured in your `Cargo.toml`.
```

Comment:
Probably nice to link the cargo docs about feature flags here (and in the analogous parts of other readmes): https://doc.rust-lang.org/cargo/reference/features.html. With that linked, you may not need the  line about "configured in `Cargo.toml`".